example/mixin/alerts.yaml

```diff
@@ -41,3 +41,13 @@ groups:
     for: 10m
     labels:
       severity: warning
+  - alert: PrometheusOperatorNotReady
+    annotations:
+      description: Prometheus operator in {{ $labels.namespace }} namespace isn't
+        ready to reconcile {{ $labels.controller }} resources.
+      summary: Prometheus operator not ready
+    expr: |
+      min by(namespace, controller) (min_over_time(prometheus_operator_ready{job="prometheus-operator"}[5m]) == 0)
```

Comment:
We can tweak this later, but maybe 5 minutes might be a bit too short?

Reply:
After more thinking maybe it should be:

```
min by(namespace, controller) (max_over_time(prometheus_operator_ready{job="prometheus-operator"}[5m]) == 0)
```